fares-data-build-tool/create-data
# feat: CFD-385 Remove requirement for end date

# Description

-   Remove requirement for end date

# Testing instructions

-   For a p2p and non-p2p ticket, create one with and without an end date
- Check that netex gets created and validated with/without an end date as expected
- Check that the fares are displayed correctly in My fares

# Type of change
